InfluxDB.Client.AspNetCore/InfluxDBClientHealthCheck.cs

```diff
@@ -0,0 +1,29 @@
+using System.Threading;
+using System.Threading.Tasks;
+using InfluxDB.Client.Api.Domain;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace InfluxDB.Client.AspNetCore
+{
+
+    public class InfluxDBClientHealthCheck : IHealthCheck
```

Comment:
Please add tests for `InfluxDBClientHealthCheck`. We want to be sure that everything works between releases.